util/jsonutil/jsonutil.go

```diff
@@ -110,3 +112,43 @@ func DropElement(extension []byte, elementNames ...string) ([]byte, error) {
 	}
 	return extension, nil
 }
+
+// Unmarshal unmarshals a byte slice into the specified data structure without performing
+// any validation on the data. An unmarshal error is returned if a non-validation error occurs.
+func Unmarshal(data []byte, v interface{}) error {
+	err := jsoniter.Config{
+		EscapeHTML:             true,
+		SortMapKeys:            true,
+		ValidateJsonRawMessage: false,
+	}.Froze().Unmarshal(data, v)
```

Comment:
Can we move the config to a package private variable? Such that the initialization and `Froze` command will be cached? Looking to duplicate the single object used in `UnmarshalValid`.